pkg/virtctl/scp/scp_test.go

```diff
@@ -0,0 +1,48 @@
+package scp_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/scp"
+)
+
+var _ = Describe("SCP", func() {
+	DescribeTable("ParseArguments", func(arg0, arg1 string, expLocal scp.LocalArgument, expRemote scp.RemoteArgument, expToRemote bool) {
+		local, remote, toRemote, err := scp.ParseTarget(arg0, arg1)
+
+		Expect(err).ToNot(HaveOccurred())
+		Expect(*local).To(Equal(expLocal))
+		Expect(*remote).To(Equal(expRemote))
```

Comment:
```suggestion
		Expect(*local).To(Equal(expLocal))
		Expect(*remote).To(Equal(expRemote))
```

nit: could use the `PointTo` matcher